npm/test-integration-newman.js

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env node
+require('shelljs/global');
+require('colors');
+
+var path = require('path'),
+    async = require('async'),
+    tmp = require('tmp');
```

Comment:
Consider using the native `dir = require('os').tmpdir()` to get a temporary directory, instead of adding a new devDependency.